testing/test_manager.py

```diff
@@ -44,6 +53,12 @@ def test_manager_receive_alarm(manager_service):
     """
     Start Manager and submit an alarm event using AMQP. Verify that the log output matches the expectations.
     """
+    service = WebinterfaceServiceWrapper()
+    service.run()
+
+    testing.test_webinterface.test_webinterface_create_worker(service)
+    testing.test_webinterface.test_webinterface_create_zone(service)
+    testing.test_webinterface.test_webinterface_create_sensor(service)
```

Comment:
Ok, but we **must not** call `test_` functions for provisioning database objects from within other `test_` functions, because this will have unintended side effects.

If you have the need to share corresponding code between multiple `test_` functions, please refactor the setup code to a utility function or, even better, a pytest fixture.